full_utility_rates/California/City of Bakersfield - 57/cob-2017-06-01.owrs

```diff
@@ -0,0 +1,33 @@
+---
+metadata:
+  effective_date: 2016-07-01
+  utility_name: "City of Bakersfield"
+  bill_frequency: "monthly"
+  # Within city limits. Rate are different in Fairhaven & Unincorporated Areas
+rate_structure:
+  RESIDENTIAL_SINGLE:
+    service_charge:
+      depends_on: meter_size
+      values:
+         5/8"  : 9.88
+         3/4"  : 9.88
+         1"    : 15.06
+         1_1/2": 22.98
+         2"    : 31.28
+         3"    : 57.80
+         4"    : 84.02
+         6"    : 151.00
+         8"    : 232.12
+         10"   : 319.95
```

Comment:
Need to add bill structure and commodity_charge if there is one such as 

if there is a commodity_charge
bill: "commodity_charge+service_charge"

or if only service_charge
bill: "service_charge"